server/views/pages/bookedtoday/arrivals/searchforexisting/search/searchForExistingRecord.njk

```diff
@@ -11,8 +11,7 @@
                     valid PNC or prison number.</p>
                 <p>Check their personal details are correct and add a PNC or prison number if you have one. This information will be
                     used to search for an existing prisoner record.</p>
-                <h2 class="govuk-heading-m">Information from the Person Escort Record</h2>
-                <h3 class="govuk-heading-m">Personal details</h3>
```

Comment:
Removing heading as discussed as this data won't be from the PER if the user modifies any data on the page